README.md

```diff
@@ -1,170 +1,170 @@
+
+```swift
+let kClientID = "2a858956-70de-42b9-b5db-d566eb1fb820"
+```
+
+## Step 2: (Optional) 
+
+## 2A: Register your App  
+
+This app comes pre-configured for testing.  If you would like to register your own app, please follow the steps below.
+
+To Register an app:
+1. Sign in to the [Microsoft Entra admin center](https://portal.azure.com) using either a work or school account.
+2. In the left-hand navigation pane, select the **Microsoft Entra ID** blade, and then select **App registrations**.
+3. Click on the **New registration** button at the top left of the page.
+4. On the app registration page,
+   - Name your app
+   - Under **Supported account types**, select **Accounts in any organizational directory (Any Microsoft Entra ID directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
+   - Click **Register** to finish.
+5. After the app is created, you'll land on your app management page. Take note of the **Application (client) ID** as this would be needed for the step 1B below.
+6. Click **Authentication**, and add new Redirect URI with type **Public client (mobile & desktop)**. Enter redirect URI in format: `msauth.<app.bundle.id>://auth`. Replace <app.bundle.id> with the **Bundle Identifier** for your application. 
+7. Hit the **Save** button in the top left, to save these updates.
+8. Click **Make this change for me** and then download the code sample for macOS
+
+## Step 3: Run the sample
+
+1. Click the Run Button in the top menu or go to Product from the menu tab and click Run.
+2. Once the sample app launches, click on the 'Call Microsoft Graph API' button to go through the sign in flow and see the results from Microsoft Graph.
+
+## How to add MSAL library into your existing Xcode project
+
+## Step 1: Configure your application Info.plist
+
+Add URI scheme in the  `Info.plist`. Redirect URI scheme follows the format `msauth.[app_bundle_id]`. Make sure to substitute [app_bundle_id] with the **Bundle Identifier** for your application.
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+  <dict>
+    <key>CFBundleURLSchemes</key>
+    <array>
+      <string>msauth.[app_bundle_id]</string>
+    </array>
+  </dict>
+</array>
+```
+
+## Step 2: Configure your application defaults
+
+In your app, add the `kClientID` variable with your Application (client) ID.
+
+```swift
+// For example, you can declare a client id in this way. Below ID is just a sample.
+    	
+let kClientID = "66855f8a-60cd-445e-a9bb-8cd8eadbd3fa"
+```
+
+Add variables with your Microsoft Entra authority and Microsoft Graph endpoint for your national cloud.
+
+```swift
+let kGraphEndpoint = "https://graph.microsoft.com/"
+let kAuthority = "https://login.microsoftonline.com/common"
+```
+
+Other endpoints are documented [here](https://docs.microsoft.com/en-us/graph/deployments#app-registration-and-token-service-root-endpoints). For example, to run the sample with Microsoft Entra Germany, use following:
```

Comment:
Let's figure out with Celeste's team what's expected here _before_ we merge.

Reply:
Thanks! Will keep you posted

Reply:
Celeste has told me that "Azure AD Germany" was a mistake and doesn't exist. It should've been just "Azure Germany" and should remain so. I will update the readme to say Azure Germany.

Reply:
Turns out I updated it to say Azure Germany yesterday so we're good. I have just fixed this issue in the other sample (mobile-apple-swift-objc)